app/models/users.py

```diff
@@ -0,0 +1,18 @@
+from typing import Union
+from sqlmodel import Field, SQLModel
+
+
+class Token(SQLModel):
```

Comment:
Can we have a comment explaining the purpose of Token and TokenData please, they are really great to have however it would be useful if we had documentation explaining that they are the auth JWT.